File: CSharp/DP.Domain/Samples/Strategy/DbLogger.cs
using System;

namespace DP.Domain.Samples.Strategy
{
    public class DbLogger:ILogger
    {
        public void Debug(string msg) => System.Diagnostics.Trace.WriteLine($"(Database)Debug: {msg}");
        public void Warn(string msg) => System.Diagnostics.Trace.WriteLine($"(Database)Warn: : {msg}");
        public void Error(string msg) => System.Diagnostics.Trace.WriteLine($"(Database)Error: : {msg}");
    }
}

File: Python/Samples/Strategy/DbLogger.py
from BaseLogger import BaseLogger

class DbLogger(BaseLogger):
    def debug(self,msg):
        print("(Database)Debug: " + msg)

    def warn(self,msg):
        print("(Database)Warn: " + msg)

    def error(self,msg):
        print("(Database)Error: " + msg)
